lighthouse-core/computed/main-thread-tasks.js

```diff
@@ -133,6 +138,12 @@ class MainThreadTasks {
       }
     }
 
+    while (currentTask && !Number.isFinite(currentTask.endTime)) {
```

Comment:
add this sorta comment in here? took me a second to determine what currentTask was at this point.

    // Starting from the last and bottom-most task,